app/javascript/stylesheets/application.scss

```diff
@@ -1 +1,23 @@
-@import "bootstrap"
\ No newline at end of file
+@import "bootstrap";
+@import '~stylesheets/pages.scss';
```

Comment:
https://kentwynn.com/frontend-lang/css/stop-import-and-start-use-forward-in-sass/kentwynn/22/07/2021/

Try to change to this